pydantic/errors.py

```diff
@@ -49,7 +49,7 @@ class NoneIsAllowedError(PydanticTypeError):
 
 class WrongConstantError(PydanticValueError):
     code = 'const'
-    msg_template = 'expected constant value {const!r}'
+    msg_template = 'unexpected constant value; permitted values: {allowed!r}'
```

Comment:
I think it might be confusing that `allowed` is displayed as a list.

I guess the simplest solution would be to override `__str__` and do something ugly like

```py
return 'unexpected constant value; permitted values: ' + ', '.join(repr(v) for v in self.ctx['allowed'])
```

Reply:
also, might be good to use consistent naming between the message and the context. Perhaps changed `allowed` -> `permitted`.